core/controllers/suggestion.py

```diff
@@ -495,8 +549,9 @@ def _upload_suggestion_images(request, suggestion, filenames):
     suggestion_image_context = suggestion.image_context
     # TODO(#10513) : Find a way to save the images before the suggestion is
     # created.
+    images = request.get('files')
     for filename in filenames:
-        image = request.get(filename)
+        image = images.get(filename).encode('ISO-8859-1') if images else None
```

Comment:
Why `ISO-8859-1`?

Reply:
After your suggestion I have used `base64` for encoding/decoding.